cli/wallet/multisig.go

```diff
@@ -14,26 +15,48 @@ import (
 )
 
 func signStoredTransaction(ctx *cli.Context) error {
-	wall, err := openWallet(ctx.String("wallet"))
+	c, err := paramcontext.Read(ctx.String("in"))
 	if err != nil {
 		return cli.NewExitError(err, 1)
 	}
-	defer wall.Close()
 
-	c, err := paramcontext.Read(ctx.String("in"))
+	tx, ok := c.Verifiable.(*transaction.Transaction)
+	if !ok {
+		return cli.NewExitError("verifiable item is not a transaction", 1)
+	}
+
+	if ctx.Bool("testinvoke") {
+		gctx, cancel := options.GetTimeoutContext(ctx)
+		defer cancel()
+
+		var err error // `GetRPCClient` returns specialized type.
+		cl, err := options.GetRPCClient(gctx, ctx)
+		if err != nil {
+			return cli.NewExitError(err, 1)
+		}
+		res, err := cl.InvokeScript(tx.Script, tx.Signers)
+		if err != nil {
+			return cli.NewExitError(err, 1)
+		}
+		resS, err := json.MarshalIndent(res, "", " ")
+		if err != nil {
+			return cli.NewExitError(err, 1)
+		}
+		fmt.Fprintln(ctx.App.Writer, string(resS))
+		return nil
```

Comment:
Maybe it's better to continue the signing process after testinvoke?

Reply:
It's not so important, but I would expect firstly to see the transaction itself (particularly, its script) and after that to see the results of test invocation, i.e. the result of that script. Otherwise, we have the result, but are not aware of the source of that result.